cluster/gce/config-test.sh

```diff
@@ -207,7 +207,7 @@ export ETCD_DOCKER_REPOSITORY=${TEST_ETCD_DOCKER_REPOSITORY:-}
 export ETCD_VERSION=${TEST_ETCD_VERSION:-}
 
 # Default Log level for all components in test clusters and variables to override it in specific components.
-TEST_CLUSTER_LOG_LEVEL=${TEST_CLUSTER_LOG_LEVEL:---v=4}
+TEST_CLUSTER_LOG_LEVEL=${TEST_CLUSTER_LOG_LEVEL:---v=2}
```

Comment:
this doesn't control integration tests, right?

Reply:
```
// MinVerbosity determines the minimum klog verbosity when running tests that
// involve the apiserver.  This overrides the -v value from the command line,
// i.e. -v=0 has no effect when MinVerbosity is 4 (the default).  Tests can opt
// out of this by setting MinVerbosity to zero before starting the control
// plane or choose some different minimum verbosity.
var MinVerbosity = 4
```

I think that's where this is getting set in integration

Reply:
@liggitt going to check that in the current CI job that is running. will let you know.

Reply:
I'm confident cluster scripts aren't used in integration

Reply:
updated PR, there's a few more places.

Reply:
unless verbosity level is causing us issues in e2e tests, I'd really like to not drop verbosity there... logs are the main way we can piece together issues that reproduce rarely or only in the CI env

Reply:
logs show here's how it is set.
```
+ popd
+ export KUBE_COVER=n
+ KUBE_COVER=n
+ export ARTIFACTS=/logs/artifacts
+ ARTIFACTS=/logs/artifacts
+ export KUBE_KEEP_VERBOSE_TEST_OUTPUT=y
+ KUBE_KEEP_VERBOSE_TEST_OUTPUT=y
+ export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
+ KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
+ export LOG_LEVEL=4
+ LOG_LEVEL=4
+ cd /home/prow/go/src/k8s.io/kubernetes
```